src/test/resources/EiffelSourceChangeCreatedEvent.json

```diff
@@ -1,7 +1,7 @@
 {
   "meta": {
     "type": "EiffelSourceChangeCreatedEvent",
-    "version": "1.0.0",
+    "version": "4.0.0",
```

Comment:
Shouldn't this be 3? =)

Reply:
EiffelSourceChangeCreatedEvent is available in 4.0.0